sys/cbor/cbor.c

```diff
@@ -319,10 +335,14 @@ static size_t decode_int(const cbor_stream_t *s, size_t offset, uint64_t *val)
 
     *val = 0; /* clear val first */
 
+    CBOR_ENSURE_SIZE_READ(s, offset + 1);
```

Comment:
I know we check the case here, but in other cases we don't (e.g. `[encode][decode]_bytes(...)`)